### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by fredden**
This should squash all the mutants which are detected in https://github.com/maglnet/ComposerRequireChecker/pull/388

### CODE REVIEW

File: test/fixtures/validJsonExcessiveDepth.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "foo": "bar",
+  "baz": [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[
+    "This string is at a depth of 513, which is larger than the allowed maximum of 512. 512 is the built-in default."
```
Comment (Ocramius) [inline test/fixtures/validJsonExcessiveDepth.json:4]: Wow D:

**Review by Ocramius — APPROVED** [review]
Besides CS issues, nice!

**Comment by fredden** [comment]
> Besides CS issues

Sorry! I forgot to run the full test-suite before pushing up / committing. I was too focused on just the mutation stuff it seems. I've force-pushed with the fixes applied by phpcbf.

**Comment by Ocramius** [comment]
Thanks @fredden!

**Pull request merged**